src/eigen.jl

```diff
@@ -13,6 +13,10 @@ end
     end
 end
 
+function eigvals(A::StaticMatrix{N,N}, B::StaticMatrix{N,N}; kwargs...) where N
+    return SVector{N}(eigvals(Array(A), Array(B); kwargs...))
```

Comment:
`eigvals` itself is quite slow and makes some copies internally before it finally calls `LAPACK.ggev!` so I doubt it would make a noticeable difference.

Reply:
Right. I guess what I mean is, `eigvals` immediately copies the arrays and then calls `LinearAlgebra.eigvals!`, so at least by passing the `drop_sdims` version we might avoid one more unnecessary copy. But it may not help regardless :man_shrugging: